### PR TITLE
New Presence: Rooster Teeth

### PR DESCRIPTION
**Pull request by chaos511**
closes https://github.com/PreMiD/Presences/issues/644

### CODE REVIEW

File: Rooster Teeth/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "service": "Rooster Teeth",
+    "description": {
+      "en": "Rooster Teeth is a pioneering media and entertainment company responsible for some of the biggest online series in history, such as the award-winning and longest-running web series, Red vs. Blue"
+    },
+    "url": "roosterteeth.com",
+    "version": "1.0.1",
+    "logo": "https://i.imgur.com/sxZnjmY.png",
+    "thumbnail": "https://i.imgur.com/BwJU788.png",
+    "color": "#2c2420",
+    "tags": ["video", "media","streaming"],
```
Comment (Bas950) [inline Rooster Teeth/dist/metadata.json:15]: fixes the spacing between media and streaming tags

**Comment by chaos511** [comment]
> you really don't need all your "try" and "catch"'s
Should they be Removed?

File: Rooster Teeth/dist/metadata.json (hunk 1 above)
Comment (chaos511) [reply Rooster Teeth/dist/metadata.json]: fixed

**Comment by chaos511** [comment]
i have used the prettier extension

File: Rooster Teeth/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "author": {
+      "name": "qwerty142",
+      "id": "455167347464863746"
+    },
+    "service": "Rooster Teeth",
+    "description": {
+      "en": "Rooster Teeth is a pioneering media and entertainment company responsible for some of the biggest online series in history, such as the award-winning and longest-running web series, Red vs. Blue"
+    },
+    "url": "roosterteeth.com",
+    "version": "1.0.1",
```
Comment (Bas950) [inline Rooster Teeth/dist/metadata.json:11]: version 1.0.0 on first release

**Comment by doomlerd** [comment]
and of the homepage?

**Comment by chaos511** [comment]
Capture

**Pull request merged**